merge_coda_files/merge_coda_files.py

```diff
@@ -0,0 +1,119 @@
+import argparse
+import time
+from os import path
+import json
+from dateutil.parser import isoparse
+
+from core_data_modules.cleaners import Codes
+from core_data_modules.cleaners.cleaning_utils import CleaningUtils
+from core_data_modules.traced_data import Metadata
+from core_data_modules.traced_data.io import TracedDataJsonIO, TracedDataCoda2IO
+from core_data_modules.util import IOUtils
+from core_data_modules.data_models import Scheme
+
+# opens a coding scheme
+def open_scheme(filepath):
+    with open(filepath, "r") as f:
+        firebase_map = json.load(f)
+        return Scheme.from_firebase_map(firebase_map)
+
+def get_strings_coda(key_map, code_scheme, list_td):
+    list_codeids = [code.code_id for code in code_scheme.codes]
+    for td in list_td:
+            td.append_data(
+                {new_key: code_scheme.get_code_with_id(td[old_key]["CodeID"]).string_value
+                for new_key, old_key in key_map.items()
+                if old_key in td
+                if td[old_key]["CodeID"] in list_codeids},
```

Comment:
How can you have a label which isn't in the scheme?

Reply:
I can quite clearly see that this is in a for loop. But, if I understand correctly, it’s testing that the code id of the label at td[old_key] is one of the code ids in the scheme. I still don’t understand how you’ve managed to get labels from a different scheme in td[old_key]

Reply:
I understand what you're saying but I can't remember the exact reason I added this them see if I can recreate the error I come back with an explanation